examples/example_sunspot.yaml

```diff
@@ -0,0 +1,34 @@
+# The directory the workflow will output to
+output_dir: example_output
+
+# Path to a directory containing many input subdirectories, each containing a PDB file.
+simulation_input_dir: data/test_systems
+
+# Uncomment this (and comment out the above line) to continue running the previous simulations.
+#simulation_input_dir: example_output/tasks
+
+# The simulation parameters
+simulation_config:
+  solvent_type: explicit
+  simulation_length_ns: 0.1
+  report_interval_ps: 1
+  dt_ps: 0.002
+  temperature_kelvin: 310
+  heat_bath_friction_coef: 1
+  explicit_barostat: MonteCarloAnisotropicBarostat
+
+# How many parallel tasks to run, this should generally be set to the number of GPUs available.
+num_parallel_tasks: 12
+
+# The compute settings
+
+# Uncomment this to run on Sunspot (please adjust settings appropriately 
+# and comment out the workstation compute settings)
+compute_settings:
+   name: sunspot # Run on the Polaris supercomputer @ ALCF
+   num_nodes: 1 # 4 GPUs per node, if you change this, make sure to change num_parallel_tasks
+   worker_init: "module load frameworks/2024.1; conda activate my_env"
```

Comment:
Here it's surprising to me that we don't need to load more modules, export envs, etc. Nice that it is simple!